prodsys/adapters/adapter.py

```diff
@@ -681,6 +682,7 @@ def check_states(cls, state: state_data.STATE_DATA_UNION, values):
             )
         return state
 
+    #TODO: Add validator for nodes_data
```

Comment:
Add validation and Not only comment please

Reply:
Commit 22